pkg/defaults/defaults.go

```diff
@@ -59,6 +59,7 @@ const (
 	DefaultRegistryTag          = "2.7"
 	DefaultProcTag              = "83cfe07"
 	DefaultMkimageTag           = "8.5.0"
+	DefaultSDNVersion           = "v1.0.0"
```

Comment:
can you also add this [here](https://github.com/lf-edge/eden/blob/00d2e6c817108530efe3dea4a1eaa2b5d47a0382/pkg/openevec/defaults.go#L17)?

Reply:
It is already there: https://github.com/milan-zededa/eden/blob/sdn-vm-image/pkg/openevec/defaults.go#L188

Reply:
Oh, missed that, thank you!